LongWarOfTheChosen/Localization/XComGame.rus

```diff
@@ -4666,3 +4666,281 @@ TacticalText="<Bullet/> Плазмобластер выпускает разру
 
 [SparkRifle_BM X2WeaponTemplate]
 TacticalText="<Bullet/> Элериевая фазовая пушка, разработанная специально для \"Искры\", - вершина оружейных технологий.\n<Bullet/> Выстрел из основного оружия всегда завершает ход солдата при отсутствии особых способностей.<br/><Bullet/> Плазменное оружие наносит цели 1 дополнительного урона при каждом попадании."
+
+
+; LWOTC Translated
++AbilityTreeTitles[0]="Битва Будущего"
++AbilityTreeTitles[1]="Непробиваемость"
++AbilityTreeTitles[2]="Машина войны"
++AbilityTreeTitles[3]="XCOM"
+; End Translated
+
+; LWOTC Translated
+LocLongDescription="У вас есть+<Ability:TRIGGER_BSC_AIM_BONUS/> (after reaction fire penalty) попасть ответным выстрелом из основого оружия."
+; End Translated
+
+; LWOTC Translated
+LocLongDescription="У вас есть +<Ability:TRIGGER_ADV_AIM_BONUS/> (after reaction fire penalty) Попасть ответным выстрелом из основого оружия."
+; End Translated
+
+; LWOTC Translated
+LocLongDescription="У вас есть +<Ability:TRIGGER_SUP_AIM_BONUS/> (after reaction fire penalty) Попасть ответным выстрелом из основого оружия."
+; End Translated
```

Comment:
As before, these three blocks should be replacing the existing entries under `[Hair_Trigger_LW_Bsc_Ability X2AbilityTemplate]`, `[Hair_Trigger_LW_Adv_Ability X2AbilityTemplate]`, and `[Hair_Trigger_LW_Sup_Ability X2AbilityTemplate]`.